src/main/java/org/opensearch/flowframework/indices/FlowFrameworkIndicesHandler.java

```diff
@@ -104,6 +109,15 @@ public static String getWorkflowStateMappings() throws IOException {
         return getIndexMappings(WORKFLOW_STATE_INDEX_MAPPING);
     }
 
+    /**
+     * Get master-key index mapping
+     * @return master-key index mapping
+     * @throws IOException if mapping file cannot be read correctly
+     */
+    public static String getMasterKeyMappings() throws IOException {
```

Comment:
we shouldn't need to have a separate getMasterKeyMapping? i think this should be taken care of by index.getMapping like the other indices since its established as part of the FlowFrameworkIndex enum

Reply:
This is used to create that enum entry to provide the master key mapping

Reply:
sounds good, can we just change to getConfigIndexMapping() or getConfigMapping() then?

Reply:
Ah good catch, I missed that